.github/ISSUE_TEMPLATE/documentation_issue.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Documentation issue
+about: Report issues with the Microsoft Edge documentation
```

Comment:
```suggestion
about: Report issues with the Microsoft Edge documentation content.
```
Needs a period and a sharper, more explicit contrast with reporting a "doc system bug".